presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -89,7 +89,7 @@
     private HiveCompressionCodec orcCompressionCodec = HiveCompressionCodec.GZIP;
     private boolean respectTableFormat = true;
     private boolean immutablePartitions;
-    private boolean createEmptyBucketFiles ;
```

Comment:
Nit:  try to fix trailing space in the original commit

Reply:
Fixed, thanks.

Reply:
@v-jizhang, seems the change is still in the wrong commit?